<h1 align="center">
    best-of-bioinformatics
    <br>
</h1>


## Contents

- [Assembler](#assembler) _10 projects_
- [Compression](#compression) _5 projects_
- [Copy Number Variation](#copy-number-variation) _7 projects_
- [K-mer Counter](#k-mer-counter) _4 projects_
- [LiftOver](#liftover) _4 projects_
- [MHC/HLA](#mhchla) _7 projects_
- [Microsatellite Instability](#microsatellite-instability) _4 projects_
- [Mitochondrial DNA](#mitochondrial-dna) _3 projects_
- [NGS Alignment](#ngs-alignment) _11 projects_
- [Primer Designing](#primer-designing) _3 projects_
- [QC for FASTQ](#qc-for-fastq) _5 projects_
- [RNA Quantification](#rna-quantification) _3 projects_
- [Sequence Alignment](#sequence-alignment) _7 projects_
- [Short Tandem Repeats](#short-tandem-repeats) _7 projects_
- [Structural Variant](#structural-variant) _13 projects_
- [Tools for BED/GTF/GFF](#tools-for-bedgtfgff) _5 projects_
- [Tools for FASTA/FASTQ](#tools-for-fastafastq) _5 projects_
- [Tools for SAM/BAM/CRAM](#tools-for-sambamcram) _11 projects_
- [Tools for VCF/BCF](#tools-for-vcfbcf) _7 projects_
- [Variant Annotation](#variant-annotation) _6 projects_
- [Variant Caller](#variant-caller) _14 projects_
- [Visualization](#visualization) _9 projects_
- [Workflow Language](#workflow-language) _5 projects_
- [Others](#others) _32 projects_

## Explanation
- 🥇🥈🥉&nbsp; Combined project-quality score
- ⭐️&nbsp; Star count from GitHub
- 🐣&nbsp; New project _(less than 12 months old)_
- 💤&nbsp; Inactive project _(12 months no activity)_
- 💀&nbsp; Dead project _(36 months no activity)_
- 📈📉&nbsp; Project is trending up or down
- ➕&nbsp; Project was recently added
- 👨‍💻&nbsp; Contributors count from GitHub
- 🔀&nbsp; Fork count from GitHub
- 📋&nbsp; Issue count from GitHub
- ⏱️&nbsp; Last update timestamp on package manager
- 📥&nbsp; Download count from package manager
- 📦&nbsp; Number of dependent projects

<br>

## Assembler

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/marbl/canu">canu</a></b> (🥇19 ·  ⭐ 450) - A single molecule sequence assembler for genomes large and small.</summary>

- [GitHub](https://github.com/marbl/canu) (👨‍💻 49 · 🔀 150 · 📥 31K · 📋 1.9K - 1% open · ⏱️ 31.03.2021):

	```
	git clone https://github.com/marbl/canu
	```
</details>
<details><summary><b><a href="https://github.com/bcgsc/abyss">abyss</a></b> (🥈18 ·  ⭐ 220) - Assemble large genomes using short reads.</summary>

- [GitHub](https://github.com/bcgsc/abyss) (👨‍💻 48 · 🔀 75 · 📥 26K · 📋 290 - 2% open · ⏱️ 03.04.2021):

	```
	git clone https://github.com/bcgsc/abyss
	```
</details>
<details><summary><b><a href="https://github.com/chhylp123/hifiasm">hifiasm</a></b> (🥈15 ·  ⭐ 140) - Hifiasm: a haplotype-resolved assembler for accurate Hifi reads.</summary>

- [GitHub](https://github.com/chhylp123/hifiasm) (👨‍💻 3 · 🔀 19 · 📥 420 · 📋 86 - 25% open · ⏱️ 21.03.2021):

	```
	git clone https://github.com/chhylp123/hifiasm
	```
</details>
<details><summary><b><a href="https://github.com/ruanjue/wtdbg2">wtdbg2</a></b> (🥈14 ·  ⭐ 410) - Redbean: A fuzzy Bruijn graph approach to long noisy reads assembly.</summary>

- [GitHub](https://github.com/ruanjue/wtdbg2) (👨‍💻 7 · 🔀 78 · 📥 1.3K · 📋 220 - 7% open · ⏱️ 11.12.2020):

	```
	git clone https://github.com/ruanjue/wtdbg2
	```
</details>
<details><summary><b><a href="https://github.com/Nextomics/NextDenovo">NextDenovo</a></b> (🥈14 ·  ⭐ 170) - Fast and accurate de novo assembler for long reads.</summary>

- [GitHub](https://github.com/Nextomics/NextDenovo) (👨‍💻 2 · 🔀 40 · 📥 4.5K · 📋 110 - 13% open · ⏱️ 07.02.2021):

	```
	git clone https://github.com/Nextomics/NextDenovo
	```
</details>
<details><summary><b><a href="https://github.com/GATB/minia">minia</a></b> (🥉13 ·  ⭐ 59) - Minia is a short-read assembler based on a de Bruijn graph.</summary>

- [GitHub](https://github.com/GATB/minia) (👨‍💻 10 · 🔀 9 · 📥 5.2K · 📋 24 - 20% open · ⏱️ 04.03.2021):

	```
	git clone https://github.com/GATB/minia
	```
</details>
<details><summary><b><a href="https://github.com/dzerbino/velvet">velvet</a></b> (🥉12 ·  ⭐ 230 · 💀) - Short read de novo assembler using de Bruijn graphs.</summary>

- [GitHub](https://github.com/dzerbino/velvet) (👨‍💻 15 · 🔀 83 · ⏱️ 15.08.2014):

	```
	git clone https://github.com/dzerbino/velvet
	```
</details>
<details><summary><b><a href="https://github.com/aquaskyline/SOAPdenovo2">SOAPdenovo2</a></b> (🥉11 ·  ⭐ 160) - Next generation sequencing reads de novo assembler.</summary>

- [GitHub](https://github.com/aquaskyline/SOAPdenovo2) (👨‍💻 11 · 🔀 53 · 📋 61 - 3% open · ⏱️ 21.01.2021):

	```
	git clone https://github.com/aquaskyline/SOAPdenovo2
	```
</details>
<details><summary><b><a href="https://github.com/ruanjue/smartdenovo">smartdenovo</a></b> (🥉10 ·  ⭐ 95) - Ultra-fast de novo assembler using long noisy reads.</summary>

- [GitHub](https://github.com/ruanjue/smartdenovo) (👨‍💻 6 · 🔀 26 · 📋 46 - 15% open · ⏱️ 24.02.2021):

	```
	git clone https://github.com/ruanjue/smartdenovo
	```
</details>
<details><summary><b><a href="https://github.com/PacificBiosciences/pb-assembly">pb-assembly</a></b> (🥉9 ·  ⭐ 89) - PacBio Assembly Tool Suite: Reads in Assembly out.</summary>

- [GitHub](https://github.com/PacificBiosciences/pb-assembly) (👨‍💻 6 · 🔀 37 · ⏱️ 11.09.2020):

	```
	git clone https://github.com/PacificBiosciences/pb-assembly
	```
</details>
<br>

## Compression

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/Genetalks/gtz">gtz</a></b> (🥇12 ·  ⭐ 140 · 📈) - A high performance and compression ratio compressor for genomic data, powered by..</summary>

- [GitHub](https://github.com/Genetalks/gtz) (👨‍💻 17 · 🔀 38 · 📋 29 - 27% open · ⏱️ 06.04.2021):

	```
	git clone https://github.com/Genetalks/gtz
	```
</details>
<details><summary><b><a href="https://github.com/OpenGene/repaq">repaq</a></b> (🥈11 ·  ⭐ 82) - A fast lossless FASTQ compressor with ultra-high compression ratio.</summary>

- [GitHub](https://github.com/OpenGene/repaq) (👨‍💻 3 · 🔀 15 · 📋 18 - 33% open · ⏱️ 14.06.2020):

	```
	git clone https://github.com/OpenGene/repaq
	```
</details>
<details><summary><b><a href="https://github.com/divonlan/genozip">genozip</a></b> (🥈11 ·  ⭐ 53) - Compressor for genomic files (FASTQ, SAM/BAM, VCF, FASTA, GVF, 23andMe...), up..</summary>

- [GitHub](https://github.com/divonlan/genozip) (👨‍💻 5 · 🔀 2 · 📋 4 - 25% open · ⏱️ 24.03.2021):

	```
	git clone https://github.com/divonlan/genozip
	```
</details>
<details><summary><b><a href="https://github.com/shubhamchandak94/Spring">Spring</a></b> (🥉10 ·  ⭐ 45) - FASTQ compression.</summary>

- [GitHub](https://github.com/shubhamchandak94/Spring) (🔀 14 · ⏱️ 14.11.2020):

	```
	git clone https://github.com/shubhamchandak94/Spring
	```
</details>
<details><summary><b><a href="https://github.com/refresh-bio/DSRC">DSRC</a></b> (🥉8 ·  ⭐ 45) - DSRC - DNA Sequence Reads Compressor.</summary>

- [GitHub](https://github.com/refresh-bio/DSRC) (👨‍💻 6 · 🔀 18 · 📥 330 · 📋 19 - 68% open · ⏱️ 24.06.2020):

	```
	git clone https://github.com/refresh-bio/DSRC
	```
</details>
<br>

## Copy Number Variation

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/etal/cnvkit">cnvkit</a></b> (🥇15 ·  ⭐ 280) - Copy number variant detection from targeted DNA sequencing.</summary>

- [GitHub](https://github.com/etal/cnvkit) (👨‍💻 24 · 🔀 110 · 📋 520 - 28% open · ⏱️ 05.04.2021):

	```
	git clone https://github.com/etal/cnvkit
	```
</details>
<details><summary><b><a href="https://github.com/abyzovlab/CNVnator">CNVnator</a></b> (🥈13 ·  ⭐ 140) - a tool for CNV discovery and genotyping from depth-of-coverage by mapped reads.</summary>

- [GitHub](https://github.com/abyzovlab/CNVnator) (👨‍💻 19 · 🔀 54 · 📥 12K · 📋 220 - 73% open · ⏱️ 07.07.2020):

	```
	git clone https://github.com/abyzovlab/CNVnator
	```
</details>
<details><summary><b><a href="https://github.com/WGLab/PennCNV">PennCNV</a></b> (🥈11 ·  ⭐ 55) - Copy number vaiation detection from SNP arrays.</summary>

- [GitHub](https://github.com/WGLab/PennCNV) (👨‍💻 5 · 🔀 36 · 📋 65 - 35% open · ⏱️ 15.08.2020):

	```
	git clone https://github.com/WGLab/PennCNV
	```
</details>
<details><summary><b><a href="https://github.com/BoevaLab/FREEC">FREEC</a></b> (🥉9 ·  ⭐ 90 · 💤) - Control-FREEC: Copy number and genotype annotation in whole genome and whole..</summary>

- [GitHub](https://github.com/BoevaLab/FREEC) (👨‍💻 4 · 🔀 33 · 📋 75 - 53% open · ⏱️ 14.10.2019):

	```
	git clone https://github.com/BoevaLab/FREEC
	```
</details>
<details><summary><b><a href="https://github.com/Genotek/ClassifyCNV">ClassifyCNV</a></b> (🥉9 ·  ⭐ 14 · 🐣) - ClassifyCNV: a tool for clinical annotation of copy-number variants.</summary>

- [GitHub](https://github.com/Genotek/ClassifyCNV) (🔀 4 · ⏱️ 25.02.2021):

	```
	git clone https://github.com/Genotek/ClassifyCNV
	```
</details>
<details><summary><b><a href="https://github.com/RahmanTeam/DECoN">DECoN</a></b> (🥉8 ·  ⭐ 22 · 💤) - </summary>

- [GitHub](https://github.com/RahmanTeam/DECoN) (👨‍💻 4 · 🔀 15 · 📋 19 - 84% open · ⏱️ 15.02.2019):

	```
	git clone https://github.com/RahmanTeam/DECoN
	```
</details>
<details><summary><b><a href="https://github.com/vplagnol/ExomeDepth">ExomeDepth</a></b> (🥉7 ·  ⭐ 21) - ExomeDepth R package for the detection of copy number variants in exomes and..</summary>

- [GitHub](https://github.com/vplagnol/ExomeDepth) (👨‍💻 5 · 🔀 14 · 📋 37 - 62% open · ⏱️ 05.04.2020):

	```
	git clone https://github.com/vplagnol/ExomeDepth
	```
</details>
<br>

## K-mer Counter

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/gmarcais/Jellyfish">Jellyfish</a></b> (🥇16 ·  ⭐ 290) - A fast multi-threaded k-mer counter.</summary>

- [GitHub](https://github.com/gmarcais/Jellyfish) (👨‍💻 12 · 🔀 120 · 📥 86K · 📋 150 - 46% open · ⏱️ 05.05.2020):

	```
	git clone https://github.com/gmarcais/Jellyfish
	```
</details>
<details><summary><b><a href="https://github.com/refresh-bio/KMC">KMC</a></b> (🥈12 ·  ⭐ 130) - Fast and frugal disk based k-mer counter.</summary>

- [GitHub](https://github.com/refresh-bio/KMC) (👨‍💻 12 · 🔀 40 · 📥 7.5K · 📋 150 - 42% open · ⏱️ 01.10.2020):

	```
	git clone https://github.com/refresh-bio/KMC
	```
</details>
<details><summary><b><a href="https://github.com/OpenGene/UniqueKMER">UniqueKMER</a></b> (🥉10 ·  ⭐ 21 · 🐣) - Generate unique KMERs for every contig in a FASTA file.</summary>

- [GitHub](https://github.com/OpenGene/UniqueKMER) (👨‍💻 2 · 🔀 5 · ⏱️ 28.07.2020):

	```
	git clone https://github.com/OpenGene/UniqueKMER
	```
</details>
<details><summary><b><a href="https://github.com/thegenemyers/FASTK">FASTK</a></b> (🥉8 ·  ⭐ 79 · 🐣) - A fast K-mer counter for high-fidelity shotgun datasets.</summary>

- [GitHub](https://github.com/thegenemyers/FASTK) (🔀 3 · 📋 7 - 28% open · ⏱️ 07.04.2021):

	```
	git clone https://github.com/thegenemyers/FASTK
	```
</details>
<br>

## LiftOver

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/agshumate/Liftoff">Liftoff</a></b> (🥇14 ·  ⭐ 140) - An accurate GFF3/GTF lift over pipeline.</summary>

- [GitHub](https://github.com/agshumate/Liftoff) (👨‍💻 8 · 🔀 18 · 📦 3 · 📋 59 - 10% open · ⏱️ 08.04.2021):

	```
	git clone https://github.com/agshumate/Liftoff
	```
</details>
<details><summary><b><a href="https://github.com/konstantint/pyliftover">pyliftover</a></b> (🥈10 ·  ⭐ 60 · 💤) - Pure-python implementation of UCSC liftOver genome coordinate conversion.</summary>

- [GitHub](https://github.com/konstantint/pyliftover) (👨‍💻 2 · 🔀 11 · 📦 50 · 📋 13 - 15% open · ⏱️ 25.05.2019):

	```
	git clone https://github.com/konstantint/pyliftover
	```
</details>
<details><summary><b><a href="https://github.com/jeremymcrae/liftover">liftover</a></b> (🥈10 ·  ⭐ 13) - liftover for python, made fast with cython.</summary>

- [GitHub](https://github.com/jeremymcrae/liftover) (👨‍💻 2 · 🔀 1 · 📦 2 · ⏱️ 08.03.2021):

	```
	git clone https://github.com/jeremymcrae/liftover
	```
</details>
<details><summary><b><a href="https://github.com/liguowang/CrossMap">CrossMap</a></b> (🥈10 ·  ⭐ 12) - CrossMap is a python program to lift over genome coordinates from one genome..</summary>

- [GitHub](https://github.com/liguowang/CrossMap) (👨‍💻 2 · 🔀 10 · 📦 14 · 📋 23 - 60% open · ⏱️ 14.01.2021):

	```
	git clone https://github.com/liguowang/CrossMap
	```
</details>
<br>

## MHC/HLA

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/openvax/mhcflurry">mhcflurry</a></b> (🥇19 ·  ⭐ 110) - Peptide-MHC I binding affinity prediction.</summary>

- [GitHub](https://github.com/openvax/mhcflurry) (👨‍💻 12 · 🔀 33 · 📥 44K · 📦 15 · 📋 100 - 2% open · ⏱️ 07.01.2021):

	```
	git clone https://github.com/openvax/mhcflurry
	```
</details>
<details><summary><b><a href="https://github.com/FRED-2/OptiType">OptiType</a></b> (🥈12 ·  ⭐ 120) - Precision HLA typing from next-generation sequencing data.</summary>

- [GitHub](https://github.com/FRED-2/OptiType) (👨‍💻 13 · 🔀 65 · 📋 88 - 31% open · ⏱️ 15.09.2020):

	```
	git clone https://github.com/FRED-2/OptiType
	```
</details>
<details><summary><b><a href="https://github.com/DiltheyLab/HLA-LA">HLA-LA</a></b> (🥈12 ·  ⭐ 68) - Fast HLA type inference from whole-genome data.</summary>

- [GitHub](https://github.com/DiltheyLab/HLA-LA) (👨‍💻 10 · 🔀 23 · 📋 49 - 28% open · ⏱️ 01.10.2020):

	```
	git clone https://github.com/DiltheyLab/HLA-LA
	```
</details>
<details><summary><b><a href="https://github.com/MathOnco/NeoPredPipe">NeoPredPipe</a></b> (🥉11 ·  ⭐ 60) - Neoantigens prediction pipeline for multi- or single-region vcf files using..</summary>

- [GitHub](https://github.com/MathOnco/NeoPredPipe) (👨‍💻 4 · 🔀 15 · 📋 27 - 14% open · ⏱️ 25.03.2021):

	```
	git clone https://github.com/MathOnco/NeoPredPipe
	```
</details>
<details><summary><b><a href="https://github.com/hammerlab/prohlatype">prohlatype</a></b> (🥉11 ·  ⭐ 31 · 💤) - Probabilistic HLA typing.</summary>

- [GitHub](https://github.com/hammerlab/prohlatype) (👨‍💻 2 · 🔀 3 · 📥 83 · 📋 56 - 19% open · ⏱️ 04.08.2019):

	```
	git clone https://github.com/hammerlab/prohlatype
	```
</details>
<details><summary><b><a href="https://github.com/ANHIG/IMGTHLA">IMGTHLA</a></b> (🥉9 ·  ⭐ 86) - Github for files currently published in the IPD-IMGT/HLA FTP Directory hosted at..</summary>

- [GitHub](https://github.com/ANHIG/IMGTHLA) (👨‍💻 4 · 🔀 38 · 📋 83 - 2% open · ⏱️ 03.03.2021):

	```
	git clone https://github.com/ANHIG/IMGTHLA
	```
</details>
<details><summary><b><a href="https://github.com/humanlongevity/HLA">HLA</a></b> (🥉9 ·  ⭐ 66 · 💀) - xHLA: Fast and accurate HLA typing from short read sequence data.</summary>

- [GitHub](https://github.com/humanlongevity/HLA) (👨‍💻 4 · 🔀 35 · 📋 56 - 71% open · ⏱️ 04.10.2017):

	```
	git clone https://github.com/humanlongevity/HLA
	```
</details>
<br>

## Microsatellite Instability

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/ding-lab/msisensor">msisensor</a></b> (🥇11 ·  ⭐ 100) - microsatellite instability detection using tumor only or paired tumor-normal..</summary>

- [GitHub](https://github.com/ding-lab/msisensor) (👨‍💻 5 · 🔀 46 · 📥 350 · 📋 42 - 23% open · ⏱️ 06.01.2021):

	```
	git clone https://github.com/ding-lab/msisensor
	```
</details>
<details><summary><b><a href="https://github.com/xjtu-omics/msisensor-pro">msisensor-pro</a></b> (🥈9 ·  ⭐ 30 · 📉) - Microsatellite Instability (MSI) detection using high-throughput..</summary>

- [GitHub](https://github.com/xjtu-omics/msisensor-pro) (👨‍💻 2 · 🔀 8 · 📥 21 · 📋 24 - 54% open · ⏱️ 31.03.2021):

	```
	git clone https://github.com/xjtu-omics/msisensor-pro
	```
</details>
<details><summary><b><a href="https://github.com/niu-lab/msisensor2">msisensor2</a></b> (🥉8 ·  ⭐ 53 · 💤) - Microsatellite instability (MSI) detection for tumor only data.</summary>

- [GitHub](https://github.com/niu-lab/msisensor2) (👨‍💻 3 · 🔀 12 · 📋 20 - 85% open · ⏱️ 21.11.2019):

	```
	git clone https://github.com/niu-lab/msisensor2
	```
</details>
<details><summary><b><a href="https://github.com/niu-lab/msisensor-ct">msisensor-ct</a></b> (🥉7 ·  ⭐ 5 · 🐣) - Microsatellite instability (MSI) detection for cfDNA samples.</summary>

- [GitHub](https://github.com/niu-lab/msisensor-ct) (👨‍💻 3 · 🔀 1 · ⏱️ 20.02.2021):

	```
	git clone https://github.com/niu-lab/msisensor-ct
	```
</details>
<br>

## Mitochondrial DNA

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/seppinho/haplogrep-cmd">haplogrep-cmd</a></b> (🥇15 ·  ⭐ 40) - HaploGrep - mtDNA haplogroup classification. Supporting rCRS and RSRS.</summary>

- [GitHub](https://github.com/seppinho/haplogrep-cmd) (👨‍💻 5 · 🔀 16 · 📥 6.5K · 📋 38 - 34% open · ⏱️ 18.01.2021):

	```
	git clone https://github.com/seppinho/haplogrep-cmd
	```
</details>
<details><summary><b><a href="https://github.com/mitoNGS/MToolBox">MToolBox</a></b> (🥉13 ·  ⭐ 49) - A bioinformatics pipeline to analyze mtDNA from NGS data.</summary>

- [GitHub](https://github.com/mitoNGS/MToolBox) (👨‍💻 6 · 🔀 32 · 📋 91 - 43% open · ⏱️ 14.10.2020):

	```
	git clone https://github.com/mitoNGS/MToolBox
	```
</details>
<details><summary><b><a href="https://github.com/robertopreste/HmtNote">HmtNote</a></b> (🥉8 ·  ⭐ 12 · 💤) - Human mitochondrial variants annotation using HmtVar.</summary>

- [GitHub](https://github.com/robertopreste/HmtNote) (👨‍💻 2 · ⏱️ 16.08.2019):

	```
	git clone https://github.com/robertopreste/HmtNote
	```
</details>
<br>

## NGS Alignment

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/lh3/minimap2">minimap2</a></b> (🥇19 ·  ⭐ 930) - A versatile pairwise aligner for genomic and spliced nucleotide sequences.</summary>

- [GitHub](https://github.com/lh3/minimap2) (👨‍💻 28 · 🔀 240 · 📥 44K · 📦 79 · 📋 670 - 8% open · ⏱️ 08.04.2021):

	```
	git clone https://github.com/lh3/minimap2
	```
</details>
<details><summary><b><a href="https://github.com/lh3/bwa">bwa</a></b> (🥇18 ·  ⭐ 980) - Burrow-Wheeler Aligner for short-read alignment (see minimap2 for long-read..</summary>

- [GitHub](https://github.com/lh3/bwa) (👨‍💻 31 · 🔀 440 · 📥 51K · 📋 210 - 60% open · ⏱️ 09.03.2021):

	```
	git clone https://github.com/lh3/bwa
	```
</details>
<details><summary><b><a href="https://github.com/BenLangmead/bowtie2">bowtie2</a></b> (🥇18 ·  ⭐ 370) - A fast and sensitive gapped read aligner.</summary>

- [GitHub](https://github.com/BenLangmead/bowtie2) (👨‍💻 32 · 🔀 120 · 📥 38K · 📋 290 - 30% open · ⏱️ 12.03.2021):

	```
	git clone https://github.com/BenLangmead/bowtie2
	```
</details>
<details><summary><b><a href="https://github.com/alexdobin/STAR">STAR</a></b> (🥈17 ·  ⭐ 1.1K) - RNA-seq aligner.</summary>

- [GitHub](https://github.com/alexdobin/STAR) (👨‍💻 38 · 🔀 300 · 📥 350 · 📋 1.1K - 26% open · ⏱️ 08.03.2021):

	```
	git clone https://github.com/alexdobin/STAR
	```
</details>
<details><summary><b><a href="https://github.com/bwa-mem2/bwa-mem2">bwa-mem2</a></b> (🥈16 ·  ⭐ 410) - The next version of bwa-mem.</summary>

- [GitHub](https://github.com/bwa-mem2/bwa-mem2) (👨‍💻 10 · 🔀 50 · 📥 6.1K · 📋 110 - 17% open · ⏱️ 26.03.2021):

	```
	git clone https://github.com/bwa-mem2/bwa-mem2
	```
</details>
<details><summary><b><a href="https://github.com/DaehwanKimLab/hisat2">hisat2</a></b> (🥈16 ·  ⭐ 290) - Graph-based alignment (Hierarchical Graph FM index).</summary>

- [GitHub](https://github.com/DaehwanKimLab/hisat2) (👨‍💻 26 · 🔀 79 · 📋 240 - 55% open · ⏱️ 21.12.2020):

	```
	git clone https://github.com/DaehwanKimLab/hisat2
	```
</details>
<details><summary><b><a href="https://github.com/philres/ngmlr">ngmlr</a></b> (🥉15 ·  ⭐ 220 · 💤) - NGMLR is a long-read mapper designed to align PacBio or Oxford Nanopore..</summary>

- [GitHub](https://github.com/philres/ngmlr) (👨‍💻 13 · 🔀 33 · 📥 4K · 📋 87 - 40% open · ⏱️ 11.09.2018):

	```
	git clone https://github.com/philres/ngmlr
	```
</details>
<details><summary><b><a href="https://github.com/PacificBiosciences/blasr">blasr</a></b> (🥉14 ·  ⭐ 120) - BLASR: The PacBio long read aligner.</summary>

- [GitHub](https://github.com/PacificBiosciences/blasr) (👨‍💻 19 · 🔀 75 · ⏱️ 18.02.2021):

	```
	git clone https://github.com/PacificBiosciences/blasr
	```
</details>
<details><summary><b><a href="https://github.com/DaehwanKimLab/tophat">tophat</a></b> (🥉12 ·  ⭐ 88 · 💤) - Spliced read mapper for RNA-Seq.</summary>

- [GitHub](https://github.com/DaehwanKimLab/tophat) (👨‍💻 25 · 🔀 35 · 📋 46 - 76% open · ⏱️ 24.05.2018):

	```
	git clone https://github.com/DaehwanKimLab/tophat
	```
</details>
<details><summary><b><a href="https://github.com/brentp/bwa-meth">bwa-meth</a></b> (🥉12 ·  ⭐ 82) - fast and accurate alignment of BS-Seq reads using bwa-mem and a 3-letter genome.</summary>

- [GitHub](https://github.com/brentp/bwa-meth) (👨‍💻 10 · 🔀 39 · 📋 56 - 21% open · ⏱️ 19.02.2021):

	```
	git clone https://github.com/brentp/bwa-meth
	```
</details>
<details><summary><b><a href="https://github.com/PacificBiosciences/pbmm2">pbmm2</a></b> (🥉9 ·  ⭐ 77) - A minimap2 frontend for PacBio native data formats.</summary>

- [GitHub](https://github.com/PacificBiosciences/pbmm2) (👨‍💻 10 · 🔀 16 · ⏱️ 22.09.2020):

	```
	git clone https://github.com/PacificBiosciences/pbmm2
	```
</details>
<br>

## Primer Designing

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/libnano/primer3-py">primer3-py</a></b> (🥇15 ·  ⭐ 84) - Simple oligo analysis and primer design.</summary>

- [GitHub](https://github.com/libnano/primer3-py) (👨‍💻 4 · 🔀 26 · 📦 88 · 📋 34 - 23% open · ⏱️ 10.05.2020):

	```
	git clone https://github.com/libnano/primer3-py
	```
</details>
<details><summary><b><a href="https://github.com/primer3-org/primer3">primer3</a></b> (🥉11 ·  ⭐ 100) - Primer3 is a command line tool to select primers for polymerase chain reaction..</summary>

- [GitHub](https://github.com/primer3-org/primer3) (👨‍💻 7 · 🔀 33 · 📥 320 · 📋 35 - 31% open · ⏱️ 03.05.2020):

	```
	git clone https://github.com/primer3-org/primer3
	```
</details>
<details><summary><b><a href="https://github.com/quwubin/MFEprimer-3.0">MFEprimer-3.0</a></b> (🥉9 ·  ⭐ 18 · 💤) - PCR Primer Quality Control.</summary>

- [GitHub](https://github.com/quwubin/MFEprimer-3.0) (🔀 7 · 📥 1.7K · 📋 6 - 16% open · ⏱️ 17.10.2019):

	```
	git clone https://github.com/quwubin/MFEprimer-3.0
	```
</details>
<br>

## QC for FASTQ

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/marcelm/cutadapt">cutadapt</a></b> (🥇19 ·  ⭐ 340 · 📈) - Cutadapt removes adapter sequences from sequencing reads.</summary>

- [GitHub](https://github.com/marcelm/cutadapt) (👨‍💻 26 · 🔀 99 · 📥 58 · 📦 190 · 📋 480 - 3% open · ⏱️ 30.03.2021):

	```
	git clone https://github.com/marcelm/cutadapt
	```
</details>
<details><summary><b><a href="https://github.com/OpenGene/fastp">fastp</a></b> (🥈15 ·  ⭐ 980) - An ultra-fast all-in-one FASTQ preprocessor..</summary>

- [GitHub](https://github.com/OpenGene/fastp) (👨‍💻 17 · 🔀 210 · 📋 300 - 51% open · ⏱️ 14.10.2020):

	```
	git clone https://github.com/OpenGene/fastp
	```
</details>
<details><summary><b><a href="https://github.com/agordon/fastx_toolkit">fastx_toolkit</a></b> (🥉13 ·  ⭐ 120 · 💀) - FASTA/FASTQ pre-processing programs.</summary>

- [GitHub](https://github.com/agordon/fastx_toolkit) (👨‍💻 3 · 🔀 53 · 📥 69K · 📋 18 - 38% open · ⏱️ 14.08.2017):

	```
	git clone https://github.com/agordon/fastx_toolkit
	```
</details>
<details><summary><b><a href="https://github.com/s-andrews/FastQC">FastQC</a></b> (🥉12 ·  ⭐ 170) - A quality control analysis tool for high throughput sequencing data.</summary>

- [GitHub](https://github.com/s-andrews/FastQC) (👨‍💻 7 · 🔀 44 · 📋 68 - 35% open · ⏱️ 13.01.2021):

	```
	git clone https://github.com/s-andrews/FastQC
	```
</details>
<details><summary><b><a href="https://github.com/timflutre/trimmomatic">trimmomatic</a></b> (🥉6 ·  ⭐ 99 · 💀) - Read trimming tool for Illumina NGS data.</summary>

- [GitHub](https://github.com/timflutre/trimmomatic) (👨‍💻 2 · 🔀 68 · 📋 17 - 23% open · ⏱️ 10.03.2015):

	```
	git clone https://github.com/timflutre/trimmomatic
	```
</details>
<br>

## RNA Quantification

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/pachterlab/kallisto">kallisto</a></b> (🥇17 ·  ⭐ 410 · 💤) - Near-optimal RNA-Seq quantification.</summary>

- [GitHub](https://github.com/pachterlab/kallisto) (👨‍💻 11 · 🔀 130 · 📥 69K · 📋 240 - 45% open · ⏱️ 16.02.2020):

	```
	git clone https://github.com/pachterlab/kallisto
	```
</details>
<details><summary><b><a href="https://github.com/gpertea/stringtie">stringtie</a></b> (🥇17 ·  ⭐ 200) - Transcript assembly and quantification for RNA-Seq.</summary>

- [GitHub](https://github.com/gpertea/stringtie) (👨‍💻 9 · 🔀 41 · 📥 1.6K · 📋 320 - 45% open · ⏱️ 09.03.2021):

	```
	git clone https://github.com/gpertea/stringtie
	```
</details>
<details><summary><b><a href="https://github.com/comprna/SUPPA">SUPPA</a></b> (🥉11 ·  ⭐ 120) - SUPPA2: Fast quantification of differential splicing.</summary>

- [GitHub](https://github.com/comprna/SUPPA) (👨‍💻 10 · 🔀 43 · 📋 110 - 18% open · ⏱️ 03.11.2020):

	```
	git clone https://github.com/comprna/SUPPA
	```
</details>
<br>

## Sequence Alignment

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/bbuchfink/diamond">diamond</a></b> (🥇19 ·  ⭐ 500) - Accelerated BLAST compatible local sequence aligner.</summary>

- [GitHub](https://github.com/bbuchfink/diamond) (👨‍💻 17 · 🔀 120 · 📥 240K · 📋 400 - 10% open · ⏱️ 08.04.2021):

	```
	git clone https://github.com/bbuchfink/diamond
	```
</details>
<details><summary><b><a href="https://github.com/icebert/pblat">pblat</a></b> (🥈8 ·  ⭐ 32) - parallelized blat with multi-threads support.</summary>

- [GitHub](https://github.com/icebert/pblat) (👨‍💻 5 · 🔀 10 · 📋 14 - 21% open · ⏱️ 14.06.2020):

	```
	git clone https://github.com/icebert/pblat
	```
</details>
<details><summary><b><a href="https://github.com/djhshih/blat">blat</a></b> (🥈6 ·  ⭐ 10 · 💤) - BLAST-Like Alignment Tool.</summary>

- [GitHub](https://github.com/djhshih/blat) (🔀 5 · ⏱️ 11.01.2019):

	```
	git clone https://github.com/djhshih/blat
	```
</details>
<details><summary><b><a href="https://github.com/GSLBiotech/clustal-omega">clustal-omega</a></b> (🥈6 ·  ⭐ 4 · 💀) - Multiple sequence aligner for protein and DNA/RNA.</summary>

- [GitHub](https://github.com/GSLBiotech/clustal-omega) (👨‍💻 2 · 🔀 1 · ⏱️ 02.01.2018):

	```
	git clone https://github.com/GSLBiotech/clustal-omega
	```
</details>
<details><summary><b><a href="https://github.com/GSLBiotech/t-coffee">t-coffee</a></b> (🥈6 ·  ⭐ 3 · 💤) - Tools for multiple alignments of DNA, RNA, and Proteins.</summary>

- [GitHub](https://github.com/GSLBiotech/t-coffee) (👨‍💻 5 · ⏱️ 12.02.2020):

	```
	git clone https://github.com/GSLBiotech/t-coffee
	```
</details>
<details><summary><b><a href="https://github.com/GSLBiotech/mafft">mafft</a></b> (🥉4 ·  ⭐ 10) - Align multiple amino acid or nucleotide sequences.</summary>

- [GitHub](https://github.com/GSLBiotech/mafft) (👨‍💻 2 · 🔀 4 · ⏱️ 17.09.2020):

	```
	git clone https://github.com/GSLBiotech/mafft
	```
</details>
<details><summary><b><a href="https://github.com/GSLBiotech/muscle">muscle</a></b> (🥉3 ·  ⭐ 2) - Multiple Sequence Comparison by Log-Expectation.</summary>

- [GitHub](https://github.com/GSLBiotech/muscle) (👨‍💻 4 · 🔀 1 · ⏱️ 07.10.2020):

	```
	git clone https://github.com/GSLBiotech/muscle
	```
</details>
<br>

## Short Tandem Repeats

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/quinlan-lab/STRling">STRling</a></b> (🥇14 ·  ⭐ 33) - Detect novel (and reference) STR expansions from short-read data.</summary>

- [GitHub](https://github.com/quinlan-lab/STRling) (👨‍💻 3 · 🔀 3 · 📥 150 · 📋 31 - 35% open · ⏱️ 10.03.2021):

	```
	git clone https://github.com/quinlan-lab/STRling
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/ExpansionHunter">ExpansionHunter</a></b> (🥈13 ·  ⭐ 88) - A tool for estimating repeat sizes.</summary>

- [GitHub](https://github.com/Illumina/ExpansionHunter) (👨‍💻 14 · 🔀 17 · 📥 5.1K · 📋 53 - 41% open · ⏱️ 20.12.2020):

	```
	git clone https://github.com/Illumina/ExpansionHunter
	```
</details>
<details><summary><b><a href="https://github.com/Oshlack/STRetch">STRetch</a></b> (🥈13 ·  ⭐ 53) - Method for detecting STR expansions from short-read sequencing data.</summary>

- [GitHub](https://github.com/Oshlack/STRetch) (👨‍💻 6 · 🔀 12 · 📋 30 - 16% open · ⏱️ 25.02.2021):

	```
	git clone https://github.com/Oshlack/STRetch
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/ExpansionHunterDenovo">ExpansionHunterDenovo</a></b> (🥉11 ·  ⭐ 32) - A suite of tools for detecting expansions of short tandem..</summary>

- [GitHub](https://github.com/Illumina/ExpansionHunterDenovo) (👨‍💻 5 · 🔀 10 · 📥 480 · 📋 18 - 66% open · ⏱️ 16.02.2021):

	```
	git clone https://github.com/Illumina/ExpansionHunterDenovo
	```
</details>
<details><summary><b><a href="https://github.com/WGLab/RepeatHMM">RepeatHMM</a></b> (🥉11 ·  ⭐ 24 · 📉) - a hidden Markov model to infer simple repeats from genome sequences.</summary>

- [GitHub](https://github.com/WGLab/RepeatHMM) (👨‍💻 10 · 🔀 12 · 📥 53 · 📋 36 - 13% open · ⏱️ 19.02.2021):

	```
	git clone https://github.com/WGLab/RepeatHMM
	```
</details>
<details><summary><b><a href="https://github.com/tfwillems/HipSTR">HipSTR</a></b> (🥉10 ·  ⭐ 69) - Genotype and phase short tandem repeats using Illumina whole-genome sequencing..</summary>

- [GitHub](https://github.com/tfwillems/HipSTR) (👨‍💻 11 · 🔀 19 · 📋 71 - 8% open · ⏱️ 16.04.2020):

	```
	git clone https://github.com/tfwillems/HipSTR
	```
</details>
<details><summary><b><a href="https://github.com/bahlolab/exSTRa">exSTRa</a></b> (🥉10 ·  ⭐ 17) - Expanded STR algorithm for Illumina sequencing data.</summary>

- [GitHub](https://github.com/bahlolab/exSTRa) (👨‍💻 7 · 🔀 6 · 📥 5 · 📋 21 - 42% open · ⏱️ 30.06.2020):

	```
	git clone https://github.com/bahlolab/exSTRa
	```
</details>
<br>

## Structural Variant

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/dellytools/delly">delly</a></b> (🥇19 ·  ⭐ 250) - DELLY2: Structural variant discovery by integrated paired-end and split-read..</summary>

- [GitHub](https://github.com/dellytools/delly) (👨‍💻 9 · 🔀 100 · 📥 18K · 📋 220 - 3% open · ⏱️ 06.03.2021):

	```
	git clone https://github.com/dellytools/delly
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/manta">manta</a></b> (🥇16 ·  ⭐ 270 · 💤) - Structural variant and indel caller for mapped sequencing data.</summary>

- [GitHub](https://github.com/Illumina/manta) (👨‍💻 22 · 🔀 87 · 📥 23K · 📋 260 - 34% open · ⏱️ 17.07.2019):

	```
	git clone https://github.com/Illumina/manta
	```
</details>
<details><summary><b><a href="https://github.com/arq5x/lumpy-sv">lumpy-sv</a></b> (🥇16 ·  ⭐ 220 · 💤) - lumpy: a general probabilistic framework for structural variant discovery.</summary>

- [GitHub](https://github.com/arq5x/lumpy-sv) (👨‍💻 21 · 🔀 100 · 📥 5.7K · 📋 290 - 52% open · ⏱️ 16.04.2019):

	```
	git clone https://github.com/arq5x/lumpy-sv
	```
</details>
<details><summary><b><a href="https://github.com/fritzsedlazeck/SURVIVOR">SURVIVOR</a></b> (🥈14 ·  ⭐ 180) - Toolset for SV simulation, comparison and filtering.</summary>

- [GitHub](https://github.com/fritzsedlazeck/SURVIVOR) (👨‍💻 11 · 🔀 28 · 📋 130 - 18% open · ⏱️ 02.12.2020):

	```
	git clone https://github.com/fritzsedlazeck/SURVIVOR
	```
</details>
<details><summary><b><a href="https://github.com/fritzsedlazeck/Sniffles">Sniffles</a></b> (🥈13 ·  ⭐ 260) - Structural variation caller using third generation sequencing.</summary>

- [GitHub](https://github.com/fritzsedlazeck/Sniffles) (👨‍💻 10 · 🔀 47 · 📋 250 - 16% open · ⏱️ 18.12.2020):

	```
	git clone https://github.com/fritzsedlazeck/Sniffles
	```
</details>
<details><summary><b><a href="https://github.com/brentp/smoove">smoove</a></b> (🥈13 ·  ⭐ 150) - structural variant calling and genotyping with existing tools, but, smoothly.</summary>

- [GitHub](https://github.com/brentp/smoove) (👨‍💻 5 · 🔀 12 · 📥 3K · 📋 140 - 33% open · ⏱️ 29.01.2021):

	```
	git clone https://github.com/brentp/smoove
	```
</details>
<details><summary><b><a href="https://github.com/OpenGene/GeneFuse">GeneFuse</a></b> (🥈13 ·  ⭐ 70) - Gene fusion detection and visualization.</summary>

- [GitHub](https://github.com/OpenGene/GeneFuse) (👨‍💻 3 · 🔀 45 · 📋 28 - 85% open · ⏱️ 17.09.2020):

	```
	git clone https://github.com/OpenGene/GeneFuse
	```
</details>
<details><summary><b><a href="https://github.com/spiralgenetics/truvari">truvari</a></b> (🥉12 ·  ⭐ 89) - Structural variant toolkit for VCFs.</summary>

- [GitHub](https://github.com/spiralgenetics/truvari) (👨‍💻 11 · 🔀 23 · 📦 2 · ⏱️ 02.04.2021):

	```
	git clone https://github.com/spiralgenetics/truvari
	```
</details>
<details><summary><b><a href="https://github.com/eldariont/svim">svim</a></b> (🥉12 ·  ⭐ 83) - Structural Variant Identification Method using Long Reads.</summary>

- [GitHub](https://github.com/eldariont/svim) (🔀 7 · 📦 1 · 📋 47 - 6% open · ⏱️ 06.04.2021):

	```
	git clone https://github.com/eldariont/svim
	```
</details>
<details><summary><b><a href="https://github.com/tjiangHIT/cuteSV">cuteSV</a></b> (🥉12 ·  ⭐ 63) - Long read based human genomic structural variation detection with cuteSV.</summary>

- [GitHub](https://github.com/tjiangHIT/cuteSV) (👨‍💻 4 · 🔀 6 · 📋 24 - 8% open · ⏱️ 23.01.2021):

	```
	git clone https://github.com/tjiangHIT/cuteSV
	```
</details>
<details><summary><b><a href="https://github.com/genome/pindel">pindel</a></b> (🥉11 ·  ⭐ 130 · 💀) - Pindel can detect breakpoints of large deletions, medium sized insertions,..</summary>

- [GitHub](https://github.com/genome/pindel) (👨‍💻 30 · 🔀 82 · 📋 85 - 83% open · ⏱️ 08.05.2017):

	```
	git clone https://github.com/genome/pindel
	```
</details>
<details><summary><b><a href="https://github.com/PacificBiosciences/pbsv">pbsv</a></b> (🥉11 ·  ⭐ 51) - pbsv - PacBio structural variant (SV) calling and analysis tools.</summary>

- [GitHub](https://github.com/PacificBiosciences/pbsv) (👨‍💻 5 · 🔀 10 · ⏱️ 25.03.2021):

	```
	git clone https://github.com/PacificBiosciences/pbsv
	```
</details>
<details><summary><b><a href="https://github.com/genome/breakdancer">breakdancer</a></b> (🥉10 ·  ⭐ 91 · 💀) - SV detection from paired end reads mapping.</summary>

- [GitHub](https://github.com/genome/breakdancer) (👨‍💻 22 · 🔀 39 · 📋 25 - 84% open · ⏱️ 23.11.2015):

	```
	git clone https://github.com/genome/breakdancer
	```
</details>
<br>

## Tools for BED/GTF/GFF

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/arq5x/bedtools2">bedtools2</a></b> (🥇22 ·  ⭐ 670) - bedtools - the swiss army knife for genome arithmetic.</summary>

- [GitHub](https://github.com/arq5x/bedtools2) (👨‍💻 86 · 🔀 240 · 📥 370K · 📋 680 - 20% open · ⏱️ 22.03.2021):

	```
	git clone https://github.com/arq5x/bedtools2
	```
</details>
<details><summary><b><a href="https://github.com/daler/pybedtools">pybedtools</a></b> (🥈19 ·  ⭐ 220) - Python wrapper -- and more -- for Aaron Quinlan's BEDTools (bioinformatics..</summary>

- [GitHub](https://github.com/daler/pybedtools) (👨‍💻 34 · 🔀 80 · 📦 660 · 📋 240 - 5% open · ⏱️ 14.03.2021):

	```
	git clone https://github.com/daler/pybedtools
	```
</details>
<details><summary><b><a href="https://github.com/gpertea/gffread">gffread</a></b> (🥉17 ·  ⭐ 180) - GFF/GTF utility providing format conversions, region filtering, FASTA sequence..</summary>

- [GitHub](https://github.com/gpertea/gffread) (👨‍💻 4 · 🔀 26 · 📥 840 · 📋 68 - 38% open · ⏱️ 26.03.2021):

	```
	git clone https://github.com/gpertea/gffread
	```
</details>
<details><summary><b><a href="https://github.com/daler/gffutils">gffutils</a></b> (🥉15 ·  ⭐ 160 · 💤) - GFF and GTF file manipulation and interconversion.</summary>

- [GitHub](https://github.com/daler/gffutils) (👨‍💻 15 · 🔀 59 · 📦 170 · 📋 110 - 25% open · ⏱️ 30.12.2019):

	```
	git clone https://github.com/daler/gffutils
	```
</details>
<details><summary><b><a href="https://github.com/lh3/bedtk">bedtk</a></b> (🥉10 ·  ⭐ 100) - A simple toolset for BED files (warning: CLI may change before bedtk becomes..</summary>

- [GitHub](https://github.com/lh3/bedtk) (👨‍💻 3 · 🔀 10 · ⏱️ 04.03.2021):

	```
	git clone https://github.com/lh3/bedtk
	```
</details>
<br>

## Tools for FASTA/FASTQ

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/shenwei356/seqkit">seqkit</a></b> (🥇18 ·  ⭐ 610) - A cross-platform and ultrafast toolkit for FASTA/Q file manipulation in Golang.</summary>

- [GitHub](https://github.com/shenwei356/seqkit) (👨‍💻 5 · 🔀 90 · 📥 44K · 📋 160 - 3% open · ⏱️ 07.04.2021):

	```
	git clone https://github.com/shenwei356/seqkit
	```
</details>
<details><summary><b><a href="https://github.com/mdshw5/pyfaidx">pyfaidx</a></b> (🥇18 ·  ⭐ 310) - Efficient pythonic random access to fasta subsequences.</summary>

- [GitHub](https://github.com/mdshw5/pyfaidx) (👨‍💻 26 · 🔀 52 · 📦 450 · 📋 120 - 10% open · ⏱️ 28.02.2021):

	```
	git clone https://github.com/mdshw5/pyfaidx
	```
</details>
<details><summary><b><a href="https://github.com/lh3/seqtk">seqtk</a></b> (🥉14 ·  ⭐ 810) - Toolkit for processing sequences in FASTA/Q formats.</summary>

- [GitHub](https://github.com/lh3/seqtk) (👨‍💻 8 · 🔀 240 · 📋 130 - 24% open · ⏱️ 23.03.2021):

	```
	git clone https://github.com/lh3/seqtk
	```
</details>
<details><summary><b><a href="https://github.com/dcjones/fastq-tools">fastq-tools</a></b> (🥉10 ·  ⭐ 92) - Small utilities for working with fastq sequence files.</summary>

- [GitHub](https://github.com/dcjones/fastq-tools) (👨‍💻 5 · 🔀 36 · 📋 20 - 55% open · ⏱️ 30.10.2020):

	```
	git clone https://github.com/dcjones/fastq-tools
	```
</details>
<details><summary><b><a href="https://github.com/OpenGene/fastv">fastv</a></b> (🥉10 ·  ⭐ 73) - An ultra-fast tool for identification of SARS-CoV-2 and other microbes from..</summary>

- [GitHub](https://github.com/OpenGene/fastv) (👨‍💻 2 · 🔀 13 · 📋 11 - 36% open · ⏱️ 29.09.2020):

	```
	git clone https://github.com/OpenGene/fastv
	```
</details>
<br>

## Tools for SAM/BAM/CRAM

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/broadinstitute/picard">picard</a></b> (🥇22 ·  ⭐ 660) - A set of command line tools (in Java) for manipulating high-throughput..</summary>

- [GitHub](https://github.com/broadinstitute/picard) (👨‍💻 140 · 🔀 290 · 📥 170K · 📋 640 - 27% open · ⏱️ 06.04.2021):

	```
	git clone https://github.com/broadinstitute/picard
	```
</details>
<details><summary><b><a href="https://github.com/samtools/samtools">samtools</a></b> (🥇20 ·  ⭐ 1.1K) - Tools (written in C using htslib) for manipulating next-generation sequencing..</summary>

- [GitHub](https://github.com/samtools/samtools) (👨‍💻 79 · 🔀 450 · 📥 810K · 📋 860 - 19% open · ⏱️ 07.04.2021):

	```
	git clone https://github.com/samtools/samtools
	```
</details>
<details><summary><b><a href="https://github.com/samtools/htslib">htslib</a></b> (🥈19 ·  ⭐ 530) - C library for high-throughput sequencing data formats.</summary>

- [GitHub](https://github.com/samtools/htslib) (👨‍💻 69 · 🔀 340 · 📥 380K · 📋 520 - 23% open · ⏱️ 08.04.2021):

	```
	git clone https://github.com/samtools/htslib
	```
</details>
<details><summary><b><a href="https://github.com/biod/sambamba">sambamba</a></b> (🥈19 ·  ⭐ 410) - Tools for working with SAM/BAM data.</summary>

- [GitHub](https://github.com/biod/sambamba) (👨‍💻 23 · 🔀 75 · 📥 110K · 📋 420 - 3% open · ⏱️ 11.03.2021):

	```
	git clone https://github.com/biod/sambamba
	```
</details>
<details><summary><b><a href="https://github.com/pysam-developers/pysam">pysam</a></b> (🥈17 ·  ⭐ 490) - Pysam is a Python module for reading and manipulating SAM/BAM/VCF/BCF files. It's..</summary>

- [GitHub](https://github.com/pysam-developers/pysam) (👨‍💻 98 · 🔀 210 · 📋 700 - 27% open · ⏱️ 02.04.2021):

	```
	git clone https://github.com/pysam-developers/pysam
	```
</details>
<details><summary><b><a href="https://github.com/brentp/mosdepth">mosdepth</a></b> (🥈17 ·  ⭐ 380) - fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing.</summary>

- [GitHub](https://github.com/brentp/mosdepth) (👨‍💻 14 · 🔀 65 · 📥 72K · 📋 120 - 19% open · ⏱️ 20.11.2020):

	```
	git clone https://github.com/brentp/mosdepth
	```
</details>
<details><summary><b><a href="https://github.com/genome/bam-readcount">bam-readcount</a></b> (🥉14 ·  ⭐ 180 · 💤) - count DNA sequence reads in BAM files.</summary>

- [GitHub](https://github.com/genome/bam-readcount) (👨‍💻 13 · 🔀 67 · 📋 60 - 56% open · ⏱️ 10.02.2020):

	```
	git clone https://github.com/genome/bam-readcount
	```
</details>
<details><summary><b><a href="https://github.com/tobiasrausch/alfred">alfred</a></b> (🥉14 ·  ⭐ 86) - BAM Statistics, Feature Counting and Annotation.</summary>

- [GitHub](https://github.com/tobiasrausch/alfred) (👨‍💻 3 · 🔀 9 · 📥 800 · 📋 35 - 11% open · ⏱️ 30.03.2021):

	```
	git clone https://github.com/tobiasrausch/alfred
	```
</details>
<details><summary><b><a href="https://github.com/GregoryFaust/samblaster">samblaster</a></b> (🥉13 ·  ⭐ 160) - samblaster: a tool to mark duplicates and extract discordant and split..</summary>

- [GitHub](https://github.com/GregoryFaust/samblaster) (👨‍💻 5 · 🔀 22 · 📥 8K · 📋 42 - 19% open · ⏱️ 04.06.2020):

	```
	git clone https://github.com/GregoryFaust/samblaster
	```
</details>
<details><summary><b><a href="https://github.com/gt1/biobambam2">biobambam2</a></b> (🥉13 ·  ⭐ 85 · 💤) - Tools for early stage alignment file processing.</summary>

- [GitHub](https://github.com/gt1/biobambam2) (👨‍💻 8 · 🔀 17 · 📥 11K · 📋 70 - 38% open · ⏱️ 12.03.2019):

	```
	git clone https://github.com/gt1/biobambam2
	```
</details>
<details><summary><b><a href="https://github.com/shiquan/bamdst">bamdst</a></b> (🥉8 ·  ⭐ 89 · 💤) - a lightweight bam file depth statistical tool.</summary>

- [GitHub](https://github.com/shiquan/bamdst) (👨‍💻 4 · 🔀 36 · 📋 17 - 23% open · ⏱️ 31.10.2018):

	```
	git clone https://github.com/shiquan/bamdst
	```
</details>
<br>

## Tools for VCF/BCF

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/vcflib/vcflib">vcflib</a></b> (🥇19 ·  ⭐ 420) - C++ library and cmdline tools for parsing and manipulating VCF files.</summary>

- [GitHub](https://github.com/vcflib/vcflib) (👨‍💻 67 · 🔀 180 · 📥 1.4K · 📋 220 - 11% open · ⏱️ 17.02.2021):

	```
	git clone https://github.com/vcflib/vcflib
	```
</details>
<details><summary><b><a href="https://github.com/samtools/bcftools">bcftools</a></b> (🥇19 ·  ⭐ 370) - This is the official development repository for BCFtools. See installation..</summary>

- [GitHub](https://github.com/samtools/bcftools) (👨‍💻 46 · 🔀 160 · 📥 300K · 📋 1.2K - 15% open · ⏱️ 01.04.2021):

	```
	git clone https://github.com/samtools/bcftools
	```
</details>
<details><summary><b><a href="https://github.com/brentp/cyvcf2">cyvcf2</a></b> (🥈18 ·  ⭐ 240 · 📈) - cython + htslib == fast VCF and BCF processing.</summary>

- [GitHub](https://github.com/brentp/cyvcf2) (👨‍💻 29 · 🔀 48 · 📦 190 · 📋 140 - 22% open · ⏱️ 17.03.2021):

	```
	git clone https://github.com/brentp/cyvcf2
	```
</details>
<details><summary><b><a href="https://github.com/brentp/vcfanno">vcfanno</a></b> (🥉16 ·  ⭐ 260) - annotate a VCF with other VCFs/BEDs/tabixed files.</summary>

- [GitHub](https://github.com/brentp/vcfanno) (👨‍💻 9 · 🔀 43 · 📥 22K · 📋 120 - 23% open · ⏱️ 15.04.2020):

	```
	git clone https://github.com/brentp/vcfanno
	```
</details>
<details><summary><b><a href="https://github.com/dnanexus-rnd/GLnexus">GLnexus</a></b> (🥉16 ·  ⭐ 64) - Scalable gVCF merging and joint variant calling for population sequencing..</summary>

- [GitHub](https://github.com/dnanexus-rnd/GLnexus) (👨‍💻 7 · 🔀 22 · 📥 1.2K · 📋 84 - 47% open · ⏱️ 17.02.2021):

	```
	git clone https://github.com/dnanexus-rnd/GLnexus
	```
</details>
<details><summary><b><a href="https://github.com/vcftools/vcftools">vcftools</a></b> (🥉15 ·  ⭐ 330 · 💤) - A set of tools written in Perl and C++ for working with VCF files, such as..</summary>

- [GitHub](https://github.com/vcftools/vcftools) (👨‍💻 19 · 🔀 130 · 📥 90K · 📋 150 - 50% open · ⏱️ 10.03.2020):

	```
	git clone https://github.com/vcftools/vcftools
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/hap.py">hap.py</a></b> (🥉13 ·  ⭐ 250) - Haplotype VCF comparison tools.</summary>

- [GitHub](https://github.com/Illumina/hap.py) (👨‍💻 20 · 🔀 81 · 📥 50 · 📋 120 - 54% open · ⏱️ 18.03.2021):

	```
	git clone https://github.com/Illumina/hap.py
	```
</details>
<br>

## Variant Annotation

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/Ensembl/ensembl-vep">ensembl-vep</a></b> (🥇17 ·  ⭐ 260) - The Ensembl Variant Effect Predictor predicts the functional effects of..</summary>

- [GitHub](https://github.com/Ensembl/ensembl-vep) (👨‍💻 22 · 🔀 95 · 📋 570 - 14% open · ⏱️ 17.02.2021):

	```
	git clone https://github.com/Ensembl/ensembl-vep
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/SpliceAI">SpliceAI</a></b> (🥈11 ·  ⭐ 170) - A deep learning-based tool to identify splice variants.</summary>

- [GitHub](https://github.com/Illumina/SpliceAI) (👨‍💻 11 · 🔀 70 · 📦 4 · ⏱️ 15.05.2020):

	```
	git clone https://github.com/Illumina/SpliceAI
	```
</details>
<details><summary><b><a href="https://github.com/charite/jannovar">jannovar</a></b> (🥈11 ·  ⭐ 40 · 💤) - Annotation of VCF variants with functional impact and from databases..</summary>

- [GitHub](https://github.com/charite/jannovar) (👨‍💻 21 · 🔀 28 · 📥 200 · 📦 1 · 📋 240 - 16% open · ⏱️ 13.11.2019):

	```
	git clone https://github.com/charite/jannovar
	```
</details>
<details><summary><b><a href="https://github.com/pcingola/SnpEff">SnpEff</a></b> (🥉10 ·  ⭐ 120) - </summary>

- [GitHub](https://github.com/pcingola/SnpEff) (👨‍💻 19 · 🔀 54 · 📋 280 - 6% open · ⏱️ 08.04.2021):

	```
	git clone https://github.com/pcingola/SnpEff
	```
</details>
<details><summary><b><a href="https://github.com/WGLab/InterVar">InterVar</a></b> (🥉10 ·  ⭐ 110) - A bioinformatics software tool for clinical interpretation of genetic variants..</summary>

- [GitHub](https://github.com/WGLab/InterVar) (👨‍💻 3 · 🔀 64 · 📋 54 - 48% open · ⏱️ 01.03.2021):

	```
	git clone https://github.com/WGLab/InterVar
	```
</details>
<details><summary><b><a href="https://github.com/lgmgeo/AnnotSV">AnnotSV</a></b> (🥉8 ·  ⭐ 60) - Annotation and Ranking of Structural Variation.</summary>

- [GitHub](https://github.com/lgmgeo/AnnotSV) (👨‍💻 2 · 🔀 11 · 📋 43 - 9% open · ⏱️ 22.03.2021):

	```
	git clone https://github.com/lgmgeo/AnnotSV
	```
</details>
<br>

## Variant Caller

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/google/deepvariant">deepvariant</a></b> (🥇20 ·  ⭐ 2.2K) - DeepVariant is an analysis pipeline that uses a deep neural network to..</summary>

- [GitHub](https://github.com/google/deepvariant) (👨‍💻 19 · 🔀 540 · 📥 3.2K · 📋 390 - 0% open · ⏱️ 31.03.2021):

	```
	git clone https://github.com/google/deepvariant
	```
</details>
<details><summary><b><a href="https://github.com/freebayes/freebayes">freebayes</a></b> (🥇20 ·  ⭐ 530) - Bayesian haplotype-based genetic polymorphism discovery and genotyping.</summary>

- [GitHub](https://github.com/freebayes/freebayes) (👨‍💻 44 · 🔀 210 · 📥 4.9K · 📋 580 - 1% open · ⏱️ 15.03.2021):

	```
	git clone https://github.com/freebayes/freebayes
	```
</details>
<details><summary><b><a href="https://github.com/broadinstitute/gatk">gatk</a></b> (🥈19 ·  ⭐ 1K) - Official code repository for GATK versions 4 and up.</summary>

- [GitHub](https://github.com/broadinstitute/gatk) (👨‍💻 110 · 🔀 400 · 📥 340K · 📋 4K - 24% open · ⏱️ 08.04.2021):

	```
	git clone https://github.com/broadinstitute/gatk
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/strelka">strelka</a></b> (🥈16 ·  ⭐ 240 · 💤) - Strelka2 germline and somatic small variant caller.</summary>

- [GitHub](https://github.com/Illumina/strelka) (👨‍💻 28 · 🔀 72 · 📥 16K · 📋 190 - 44% open · ⏱️ 06.02.2019):

	```
	git clone https://github.com/Illumina/strelka
	```
</details>
<details><summary><b><a href="https://github.com/bioinform/somaticseq">somaticseq</a></b> (🥈15 ·  ⭐ 120) - An ensemble approach to accurately detect somatic mutations using SomaticSeq.</summary>

- [GitHub](https://github.com/bioinform/somaticseq) (👨‍💻 17 · 🔀 43 · 📋 58 - 1% open · ⏱️ 01.02.2021):

	```
	git clone https://github.com/bioinform/somaticseq
	```
</details>
<details><summary><b><a href="https://github.com/bioinform/neusomatic">neusomatic</a></b> (🥈13 ·  ⭐ 130 · 💤) - NeuSomatic: Deep convolutional neural networks for accurate somatic..</summary>

- [GitHub](https://github.com/bioinform/neusomatic) (👨‍💻 25 · 🔀 38 · 📋 40 - 32% open · ⏱️ 28.02.2020):

	```
	git clone https://github.com/bioinform/neusomatic
	```
</details>
<details><summary><b><a href="https://github.com/OpenGene/MutScan">MutScan</a></b> (🥈13 ·  ⭐ 110 · 💤) - Detect and visualize target mutations by scanning FastQ files directly.</summary>

- [GitHub](https://github.com/OpenGene/MutScan) (👨‍💻 4 · 🔀 35 · 📥 7 · 📋 10 - 30% open · ⏱️ 21.01.2020):

	```
	git clone https://github.com/OpenGene/MutScan
	```
</details>
<details><summary><b><a href="https://github.com/ultimatesource/denovogear">denovogear</a></b> (🥈13 ·  ⭐ 40 · 💤) - A program to detect denovo-variants using next-generation sequencing data.</summary>

- [GitHub](https://github.com/ultimatesource/denovogear) (👨‍💻 25 · 🔀 23 · 📥 920 · 📋 160 - 28% open · ⏱️ 30.03.2020):

	```
	git clone https://github.com/ultimatesource/denovogear
	```
</details>
<details><summary><b><a href="https://github.com/AstraZeneca-NGS/VarDict">VarDict</a></b> (🥉12 ·  ⭐ 140) - VarDict.</summary>

- [GitHub](https://github.com/AstraZeneca-NGS/VarDict) (👨‍💻 21 · 🔀 51 · 📋 96 - 16% open · ⏱️ 25.01.2021):

	```
	git clone https://github.com/AstraZeneca-NGS/VarDict
	```
</details>
<details><summary><b><a href="https://github.com/nygenome/lancet">lancet</a></b> (🥉10 ·  ⭐ 140) - Microassembly based somatic variant caller for NGS data.</summary>

- [GitHub](https://github.com/nygenome/lancet) (👨‍💻 2 · 🔀 32 · ⏱️ 12.08.2020):

	```
	git clone https://github.com/nygenome/lancet
	```
</details>
<details><summary><b><a href="https://github.com/dkoboldt/varscan">varscan</a></b> (🥉10 ·  ⭐ 120 · 💤) - Variant calling and somatic mutation/CNV detection for next-generation..</summary>

- [GitHub](https://github.com/dkoboldt/varscan) (👨‍💻 2 · 🔀 29 · 📥 12K · 📋 61 - 90% open · ⏱️ 25.07.2019):

	```
	git clone https://github.com/dkoboldt/varscan
	```
</details>
<details><summary><b><a href="https://github.com/broadinstitute/mutect">mutect</a></b> (🥉10 ·  ⭐ 88 · 💀) - MuTect -- Accurate and sensitive cancer mutation detection.</summary>

- [GitHub](https://github.com/broadinstitute/mutect) (👨‍💻 4 · 🔀 39 · 📥 5.9K · 📋 9 - 88% open · ⏱️ 01.06.2014):

	```
	git clone https://github.com/broadinstitute/mutect
	```
</details>
<details><summary><b><a href="https://github.com/whatshap/whatshap">whatshap</a></b> (🥉10 ·  ⭐ 83 · 🐣) - Read-based phasing of genomic variants, also called haplotype assembly.</summary>

- [GitHub](https://github.com/whatshap/whatshap) (👨‍💻 20 · 🔀 11 · 📦 4 · 📋 280 - 32% open · ⏱️ 08.04.2021):

	```
	git clone https://github.com/whatshap/whatshap
	```
</details>
<details><summary><b><a href="https://github.com/danielfan/MuSE">MuSE</a></b> (🥉6 ·  ⭐ 14 · 💀) - Somatic point mutation caller.</summary>

- [GitHub](https://github.com/danielfan/MuSE) (🔀 7 · ⏱️ 08.07.2016):

	```
	git clone https://github.com/danielfan/MuSE
	```
</details>
<br>

## Visualization

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/lmcinnes/umap">umap</a></b> (🥇25 ·  ⭐ 4.6K) - Uniform Manifold Approximation and Projection.</summary>

- [GitHub](https://github.com/lmcinnes/umap) (👨‍💻 76 · 🔀 490 · 📦 2.7K · 📋 480 - 48% open · ⏱️ 07.04.2021):

	```
	git clone https://github.com/lmcinnes/umap
	```
</details>
<details><summary><b><a href="https://github.com/igvteam/igv.js">igv.js</a></b> (🥈21 ·  ⭐ 420) - Embeddable genomic visualization component based on the Integrative Genomics..</summary>

- [GitHub](https://github.com/igvteam/igv.js) (👨‍💻 51 · 🔀 140 · 📦 91 · 📋 840 - 5% open · ⏱️ 08.04.2021):

	```
	git clone https://github.com/igvteam/igv.js
	```
</details>
<details><summary><b><a href="https://github.com/deeptools/pyGenomeTracks">pyGenomeTracks</a></b> (🥈16 ·  ⭐ 440) - python module to plot beautiful and highly customizable genome browser..</summary>

- [GitHub](https://github.com/deeptools/pyGenomeTracks) (👨‍💻 11 · 🔀 73 · 📦 6 · 📋 130 - 9% open · ⏱️ 12.11.2020):

	```
	git clone https://github.com/deeptools/pyGenomeTracks
	```
</details>
<details><summary><b><a href="https://github.com/igvteam/igv">igv</a></b> (🥈16 ·  ⭐ 420) - Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for..</summary>

- [GitHub](https://github.com/igvteam/igv) (👨‍💻 53 · 🔀 200 · 📋 760 - 9% open · ⏱️ 29.03.2021):

	```
	git clone https://github.com/igvteam/igv
	```
</details>
<details><summary><b><a href="https://github.com/ryanlayer/samplot">samplot</a></b> (🥈16 ·  ⭐ 290) - Plot structural variant signals from many BAMs and CRAMs.</summary>

- [GitHub](https://github.com/ryanlayer/samplot) (👨‍💻 9 · 🔀 34 · 📦 1 · 📋 68 - 26% open · ⏱️ 07.04.2021):

	```
	git clone https://github.com/ryanlayer/samplot
	```
</details>
<details><summary><b><a href="https://github.com/WebLogo/weblogo">weblogo</a></b> (🥈16 ·  ⭐ 93) - WebLogo 3: Sequence Logos redrawn.</summary>

- [GitHub](https://github.com/WebLogo/weblogo) (👨‍💻 15 · 🔀 28 · 📦 85 · 📋 80 - 10% open · ⏱️ 12.02.2021):

	```
	git clone https://github.com/WebLogo/weblogo
	```
</details>
<details><summary><b><a href="https://github.com/Edinburgh-Genome-Foundry/DnaFeaturesViewer">DnaFeaturesViewer</a></b> (🥉15 ·  ⭐ 260) - Python library to plot DNA sequence features (e.g. from Genbank..</summary>

- [GitHub](https://github.com/Edinburgh-Genome-Foundry/DnaFeaturesViewer) (👨‍💻 5 · 🔀 44 · 📋 48 - 50% open · ⏱️ 20.09.2020):

	```
	git clone https://github.com/Edinburgh-Genome-Foundry/DnaFeaturesViewer
	```
</details>
<details><summary><b><a href="https://github.com/joiningdata/lollipops">lollipops</a></b> (🥉13 ·  ⭐ 150 · 📈) - Lollipop-style mutation diagrams for annotating genetic variations.</summary>

- [GitHub](https://github.com/joiningdata/lollipops) (👨‍💻 4 · 🔀 51 · 📥 3.4K · 📋 47 - 4% open · ⏱️ 30.06.2020):

	```
	git clone https://github.com/joiningdata/lollipops
	```
</details>
<details><summary><b><a href="https://github.com/nspies/genomeview">genomeview</a></b> (🥉9 ·  ⭐ 110 · 💤) - An extensible python-based genomics visualization engine.</summary>

- [GitHub](https://github.com/nspies/genomeview) (🔀 12 · 📦 2 · 📋 22 - 18% open · ⏱️ 28.07.2018):

	```
	git clone https://github.com/nspies/genomeview
	```
</details>
<br>

## Workflow Language

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/snakemake/snakemake">snakemake</a></b> (🥇24 ·  ⭐ 920) - This is the development home of the workflow management system Snakemake...</summary>

- [GitHub](https://github.com/snakemake/snakemake) (👨‍💻 200 · 🔀 200 · 📦 780 · 📋 590 - 62% open · ⏱️ 07.04.2021):

	```
	git clone https://github.com/snakemake/snakemake
	```
</details>
<details><summary><b><a href="https://github.com/nextflow-io/nextflow">nextflow</a></b> (🥈22 ·  ⭐ 1.3K) - A DSL for data-driven computational pipelines.</summary>

- [GitHub](https://github.com/nextflow-io/nextflow) (👨‍💻 97 · 🔀 320 · 📥 44K · 📋 1.7K - 8% open · ⏱️ 08.04.2021):

	```
	git clone https://github.com/nextflow-io/nextflow
	```
</details>
<details><summary><b><a href="https://github.com/broadinstitute/cromwell">cromwell</a></b> (🥉19 ·  ⭐ 670) - Scientific workflow engine designed for simplicity & scalability. Trivially..</summary>

- [GitHub](https://github.com/broadinstitute/cromwell) (👨‍💻 120 · 🔀 220 · 📥 330K · 📋 2.7K - 25% open · ⏱️ 07.04.2021):

	```
	git clone https://github.com/broadinstitute/cromwell
	```
</details>
<details><summary><b><a href="https://github.com/ssadedin/bpipe">bpipe</a></b> (🥉16 ·  ⭐ 190) - Bpipe - a tool for running and managing bioinformatics pipelines.</summary>

- [GitHub](https://github.com/ssadedin/bpipe) (👨‍💻 17 · 🔀 48 · 📥 10K · 📋 240 - 59% open · ⏱️ 02.04.2021):

	```
	git clone https://github.com/ssadedin/bpipe
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/pyflow">pyflow</a></b> (🥉11 ·  ⭐ 110) - A lightweight parallel task engine.</summary>

- [GitHub](https://github.com/Illumina/pyflow) (👨‍💻 3 · 🔀 32 · 📥 2.1K · 📋 23 - 56% open · ⏱️ 01.07.2020):

	```
	git clone https://github.com/Illumina/pyflow
	```
</details>
<br>

## Others

<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back to top"></a>

<details><summary><b><a href="https://github.com/biopython/biopython">biopython</a></b> (🥇24 ·  ⭐ 2.7K) - Official git repository for Biopython (originally converted from CVS).</summary>

- [GitHub](https://github.com/biopython/biopython) (👨‍💻 370 · 🔀 1.2K · 📦 6.8K · 📋 1.2K - 27% open · ⏱️ 08.04.2021):

	```
	git clone https://github.com/biopython/biopython
	```
</details>
<details><summary><b><a href="https://github.com/ewels/MultiQC">MultiQC</a></b> (🥇22 ·  ⭐ 710) - Aggregate results from bioinformatics analyses across many samples into a..</summary>

- [GitHub](https://github.com/ewels/MultiQC) (👨‍💻 150 · 🔀 360 · 📥 44 · 📦 140 · 📋 950 - 15% open · ⏱️ 01.04.2021):

	```
	git clone https://github.com/ewels/MultiQC
	```
</details>
<details><summary><b><a href="https://github.com/COMBINE-lab/salmon">salmon</a></b> (🥇20 ·  ⭐ 460) - Highly-accurate & wicked fast transcript-level quantification from RNA-seq reads..</summary>

- [GitHub](https://github.com/COMBINE-lab/salmon) (👨‍💻 42 · 🔀 120 · 📥 79K · 📋 570 - 40% open · ⏱️ 26.11.2020):

	```
	git clone https://github.com/COMBINE-lab/salmon
	```
</details>
<details><summary><b><a href="https://github.com/bioconda/bioconda-recipes">bioconda-recipes</a></b> (🥇19 ·  ⭐ 1.3K) - Conda recipes for the bioconda channel.</summary>

- [GitHub](https://github.com/bioconda/bioconda-recipes) (👨‍💻 1.4K · 🔀 1.8K · 📋 1.4K - 19% open · ⏱️ 08.04.2021):

	```
	git clone https://github.com/bioconda/bioconda-recipes
	```
</details>
<details><summary><b><a href="https://github.com/CGATOxford/UMI-tools">UMI-tools</a></b> (🥇19 ·  ⭐ 320) - Tools for handling Unique Molecular Identifiers in NGS data sets.</summary>

- [GitHub](https://github.com/CGATOxford/UMI-tools) (👨‍💻 29 · 🔀 160 · 📥 3.4K · 📋 330 - 10% open · ⏱️ 05.02.2021):

	```
	git clone https://github.com/CGATOxford/UMI-tools
	```
</details>
<details><summary><b><a href="https://github.com/shenwei356/csvtk">csvtk</a></b> (🥈18 ·  ⭐ 570) - A cross-platform, efficient and practical CSV/TSV toolkit in Golang.</summary>

- [GitHub](https://github.com/shenwei356/csvtk) (👨‍💻 8 · 🔀 59 · 📥 11K · 📋 120 - 8% open · ⏱️ 08.04.2021):

	```
	git clone https://github.com/shenwei356/csvtk
	```
</details>
<details><summary><b><a href="https://github.com/torognes/vsearch">vsearch</a></b> (🥈16 ·  ⭐ 450) - Versatile open-source tool for microbiome analysis.</summary>

- [GitHub](https://github.com/torognes/vsearch) (👨‍💻 14 · 🔀 100 · 📥 42K · 📋 410 - 11% open · ⏱️ 29.03.2021):

	```
	git clone https://github.com/torognes/vsearch
	```
</details>
<details><summary><b><a href="https://github.com/BioContainers/containers">containers</a></b> (🥈16 ·  ⭐ 440) - Bioinformatics containers.</summary>

- [GitHub](https://github.com/BioContainers/containers) (👨‍💻 51 · 🔀 170 · 📥 18 · 📋 210 - 13% open · ⏱️ 29.03.2021):

	```
	git clone https://github.com/BioContainers/containers
	```
</details>
<details><summary><b><a href="https://github.com/Gaius-Augustus/Augustus">Augustus</a></b> (🥈16 ·  ⭐ 130) - Genome annotation with AUGUSTUS.</summary>

- [GitHub](https://github.com/Gaius-Augustus/Augustus) (👨‍💻 31 · 🔀 64 · 📥 2.8K · 📋 180 - 55% open · ⏱️ 08.04.2021):

	```
	git clone https://github.com/Gaius-Augustus/Augustus
	```
</details>
<details><summary><b><a href="https://github.com/macs3-project/MACS">MACS</a></b> (🥈15 ·  ⭐ 480) - MACS -- Model-based Analysis of ChIP-Seq.</summary>

- [GitHub](https://github.com/macs3-project/MACS) (👨‍💻 18 · 🔀 210 · 📋 380 - 51% open · ⏱️ 11.03.2021):

	```
	git clone https://github.com/macs3-project/MACS
	```
</details>
<details><summary><b><a href="https://github.com/bioinformatics-centre/kaiju">kaiju</a></b> (🥈15 ·  ⭐ 150) - Fast taxonomic classification of metagenomic sequencing reads using a protein..</summary>

- [GitHub](https://github.com/bioinformatics-centre/kaiju) (👨‍💻 10 · 🔀 45 · 📥 1.4K · 📋 170 - 10% open · ⏱️ 12.03.2021):

	```
	git clone https://github.com/bioinformatics-centre/kaiju
	```
</details>
<details><summary><b><a href="https://github.com/exomiser/Exomiser">Exomiser</a></b> (🥈15 ·  ⭐ 78 · 💤) - A Tool to Annotate and Prioritize Exome Variants.</summary>

- [GitHub](https://github.com/exomiser/Exomiser) (👨‍💻 15 · 🔀 35 · 📥 7.4K · 📋 350 - 29% open · ⏱️ 01.10.2019):

	```
	git clone https://github.com/exomiser/Exomiser
	```
</details>
<details><summary><b><a href="https://github.com/ncbi/sra-tools">sra-tools</a></b> (🥈14 ·  ⭐ 530) - SRA Tools.</summary>

- [GitHub](https://github.com/ncbi/sra-tools) (👨‍💻 33 · 🔀 160 · 📋 470 - 11% open · ⏱️ 31.03.2021):

	```
	git clone https://github.com/ncbi/sra-tools
	```
</details>
<details><summary><b><a href="https://github.com/ParBLiSS/FastANI">FastANI</a></b> (🥈14 ·  ⭐ 170) - Fast Whole-Genome Similarity (ANI) Estimation.</summary>

- [GitHub](https://github.com/ParBLiSS/FastANI) (👨‍💻 10 · 🔀 32 · 📥 3.6K · 📋 77 - 33% open · ⏱️ 30.09.2020):

	```
	git clone https://github.com/ParBLiSS/FastANI
	```
</details>
<details><summary><b><a href="https://github.com/WGLab/Phen2Gene">Phen2Gene</a></b> (🥈14 ·  ⭐ 23) - Phenotype driven gene prioritization for HPO.</summary>

- [GitHub](https://github.com/WGLab/Phen2Gene) (👨‍💻 14 · 🔀 8 · 📥 750 · 📋 8 - 12% open · ⏱️ 22.03.2021):

	```
	git clone https://github.com/WGLab/Phen2Gene
	```
</details>
<details><summary><b><a href="https://github.com/counsyl/hgvs">hgvs</a></b> (🥉13 ·  ⭐ 150 · 💤) - HGVS variant name parsing and generation.</summary>

- [GitHub](https://github.com/counsyl/hgvs) (👨‍💻 15 · 🔀 58 · 📦 6 · 📋 33 - 39% open · ⏱️ 19.03.2020):

	```
	git clone https://github.com/counsyl/hgvs
	```
</details>
<details><summary><b><a href="https://github.com/jsh58/Genrich">Genrich</a></b> (🥉13 ·  ⭐ 100) - Detecting sites of genomic enrichment.</summary>

- [GitHub](https://github.com/jsh58/Genrich) (🔀 18 · 📋 70 - 5% open · ⏱️ 16.03.2021):

	```
	git clone https://github.com/jsh58/Genrich
	```
</details>
<details><summary><b><a href="https://github.com/odelaneau/shapeit4">shapeit4</a></b> (🥉13 ·  ⭐ 55) - Segmented HAPlotype Estimation and Imputation Tool.</summary>

- [GitHub](https://github.com/odelaneau/shapeit4) (👨‍💻 3 · 🔀 10 · 📋 49 - 53% open · ⏱️ 11.03.2021):

	```
	git clone https://github.com/odelaneau/shapeit4
	```
</details>
<details><summary><b><a href="https://github.com/EddyRivasLab/hmmer">hmmer</a></b> (🥉12 ·  ⭐ 160) - HMMER: biological sequence analysis using profile HMMs.</summary>

- [GitHub](https://github.com/EddyRivasLab/hmmer) (👨‍💻 18 · 🔀 43 · 📋 190 - 4% open · ⏱️ 27.11.2020):

	```
	git clone https://github.com/EddyRivasLab/hmmer
	```
</details>
<details><summary><b><a href="https://github.com/schatzlab/genomescope">genomescope</a></b> (🥉12 ·  ⭐ 120 · 💀) - Fast genome analysis from unassembled short reads.</summary>

- [GitHub](https://github.com/schatzlab/genomescope) (👨‍💻 9 · 🔀 28 · 📋 51 - 60% open · ⏱️ 15.02.2018):

	```
	git clone https://github.com/schatzlab/genomescope
	```
</details>
<details><summary><b><a href="https://github.com/PharmGKB/PharmCAT">PharmCAT</a></b> (🥉12 ·  ⭐ 42) - The Pharmacogenomic Clinical Annotation Tool.</summary>

- [GitHub](https://github.com/PharmGKB/PharmCAT) (👨‍💻 15 · 🔀 18 · 📥 1.1K · 📋 45 - 31% open · ⏱️ 09.09.2020):

	```
	git clone https://github.com/PharmGKB/PharmCAT
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/GTCtoVCF">GTCtoVCF</a></b> (🥉11 ·  ⭐ 26) - Script to convert GTC/BPM files to VCF.</summary>

- [GitHub](https://github.com/Illumina/GTCtoVCF) (👨‍💻 10 · 🔀 19 · 📋 38 - 26% open · ⏱️ 07.04.2020):

	```
	git clone https://github.com/Illumina/GTCtoVCF
	```
</details>
<details><summary><b><a href="https://github.com/mutalyzer/mutalyzer">mutalyzer</a></b> (🥉10 ·  ⭐ 84) - HGVS variant nomenclature checker.</summary>

- [GitHub](https://github.com/mutalyzer/mutalyzer) (👨‍💻 9 · 🔀 20 · 📋 320 - 38% open · ⏱️ 15.03.2021):

	```
	git clone https://github.com/mutalyzer/mutalyzer
	```
</details>
<details><summary><b><a href="https://github.com/snugel/cas-offinder">cas-offinder</a></b> (🥉10 ·  ⭐ 54) - An ultrafast and versatile algorithm that searches for potential off-..</summary>

- [GitHub](https://github.com/snugel/cas-offinder) (👨‍💻 2 · 🔀 17 · 📥 240 · 📋 19 - 5% open · ⏱️ 23.02.2021):

	```
	git clone https://github.com/snugel/cas-offinder
	```
</details>
<details><summary><b><a href="https://github.com/lh3/bioawk">bioawk</a></b> (🥉9 ·  ⭐ 460 · 💀) - BWK awk modified for biological data.</summary>

- [GitHub](https://github.com/lh3/bioawk) (👨‍💻 7 · 🔀 97 · 📋 28 - 60% open · ⏱️ 11.09.2017):

	```
	git clone https://github.com/lh3/bioawk
	```
</details>
<details><summary><b><a href="https://github.com/dpryan79/ChromosomeMappings">ChromosomeMappings</a></b> (🥉9 ·  ⭐ 180) - This repository contains chromosome/contig name mappings between..</summary>

- [GitHub](https://github.com/dpryan79/ChromosomeMappings) (👨‍💻 13 · 🔀 48 · 📋 7 - 14% open · ⏱️ 19.11.2020):

	```
	git clone https://github.com/dpryan79/ChromosomeMappings
	```
</details>
<details><summary><b><a href="https://github.com/rmhubley/RepeatMasker">RepeatMasker</a></b> (🥉9 ·  ⭐ 78) - RepeatMasker is a program that screens DNA sequences for interspersed..</summary>

- [GitHub](https://github.com/rmhubley/RepeatMasker) (👨‍💻 4 · 🔀 25 · 📋 96 - 31% open · ⏱️ 31.03.2021):

	```
	git clone https://github.com/rmhubley/RepeatMasker
	```
</details>
<details><summary><b><a href="https://github.com/Illumina/SMNCopyNumberCaller">SMNCopyNumberCaller</a></b> (🥉9 ·  ⭐ 27) - A copy number caller for SMN1 and SMN2 to enable SMA diagnosis..</summary>

- [GitHub](https://github.com/Illumina/SMNCopyNumberCaller) (👨‍💻 3 · 🔀 6 · ⏱️ 11.06.2020):

	```
	git clone https://github.com/Illumina/SMNCopyNumberCaller
	```
</details>
<details><summary><b><a href="https://github.com/samtools/tabix">tabix</a></b> (🥉8 ·  ⭐ 82 · 💀) - Note: tabix and bgzip binaries are now part of the HTSlib project.</summary>

- [GitHub](https://github.com/samtools/tabix) (👨‍💻 4 · 🔀 38 · 📋 13 - 53% open · ⏱️ 16.12.2013):

	```
	git clone https://github.com/samtools/tabix
	```
</details>
<details><summary><b><a href="https://github.com/babelomics/SMAca">SMAca</a></b> (🥉8 ·  ⭐ 9) - SMN1 copy-number and sequence variant analysis from next generation sequencing data.</summary>

- [GitHub](https://github.com/babelomics/SMAca) (👨‍💻 8 · 🔀 1 · 📋 2 - 50% open · ⏱️ 16.10.2020):

	```
	git clone https://github.com/babelomics/SMAca
	```
</details>
<details><summary><b><a href="https://github.com/biothings/biothings_client.py">biothings_client.py</a></b> (🥉7 ·  ⭐ 7) - This is a generic Python client for BioThings APIs.</summary>

- [GitHub](https://github.com/biothings/biothings_client.py) (👨‍💻 7 · 🔀 5 · 📋 16 - 12% open · ⏱️ 06.04.2021):

	```
	git clone https://github.com/biothings/biothings_client.py
	```
</details>
<details><summary><b><a href="https://github.com/lh3/tabtk">tabtk</a></b> (🥉6 ·  ⭐ 53 · 💀) - Toolkit for processing TAB-delimited format.</summary>

- [GitHub](https://github.com/lh3/tabtk) (🔀 12 · ⏱️ 11.08.2016):

	```
	git clone https://github.com/lh3/tabtk
	```
</details>
