## ğŸ“ˆ Trending Up

_Projects that have a higher project-quality score compared to the last update. There might be a variety of reasons, such as increased downloads or code activity._

- <b><a href="https://github.com/ewels/MultiQC">MultiQC</a></b> (ğŸ¥‡20 Â·  â­ 710 Â· ğŸ“ˆ) - Aggregate results from bioinformatics analyses across many samples into a..

## â• Added Projects

_Projects that were recently added to this best-of list._

- <b><a href="https://github.com/ryanlayer/samplot">samplot</a></b> (ğŸ¥ˆ16 Â·  â­ 290 Â· â•) - Plot structural variant signals from many BAMs and CRAMs.
- <b><a href="https://github.com/arq5x/lumpy-sv">lumpy-sv</a></b> (ğŸ¥‡16 Â·  â­ 210 Â· ğŸ’¤) - lumpy: a general probabilistic framework for structural variant discovery.
- <b><a href="https://github.com/fritzsedlazeck/SURVIVOR">SURVIVOR</a></b> (ğŸ¥ˆ14 Â·  â­ 180 Â· â•) - Toolset for SV simulation, comparison and filtering.
- <b><a href="https://github.com/tobiasrausch/alfred">alfred</a></b> (ğŸ¥‰14 Â·  â­ 82 Â· â•) - BAM Statistics, Feature Counting and Annotation.
- <b><a href="https://github.com/abyzovlab/CNVnator">CNVnator</a></b> (ğŸ¥ˆ13 Â·  â­ 140 Â· â•) - a tool for CNV discovery and genotyping from depth-of-coverage by mapped..
- <b><a href="https://github.com/refresh-bio/KMC">KMC</a></b> (ğŸ¥‰12 Â·  â­ 130 Â· â•) - Fast and frugal disk based k-mer counter.
- <b><a href="https://github.com/spiralgenetics/truvari">truvari</a></b> (ğŸ¥‰12 Â·  â­ 86 Â· â•) - Structural variant toolkit for VCFs.
- <b><a href="https://github.com/DiltheyLab/HLA-LA">HLA-LA</a></b> (ğŸ¥ˆ12 Â·  â­ 68 Â· â•) - Fast HLA type inference from whole-genome data.
- <b><a href="https://github.com/hammerlab/prohlatype">prohlatype</a></b> (ğŸ¥‰11 Â·  â­ 31 Â· ğŸ’¤) - Probabilistic HLA typing.
- <b><a href="https://github.com/nygenome/lancet">lancet</a></b> (ğŸ¥‰10 Â·  â­ 130 Â· â•) - Microassembly based somatic variant caller for NGS data.
- <b><a href="https://github.com/ruanjue/smartdenovo">smartdenovo</a></b> (ğŸ¥‰10 Â·  â­ 92 Â· â•) - Ultra-fast de novo assembler using long noisy reads.
- <b><a href="https://github.com/Illumina/SMNCopyNumberCaller">SMNCopyNumberCaller</a></b> (ğŸ¥‰10 Â·  â­ 26 Â· â•) - A copy number caller for SMN1 and SMN2 to enable SMA..
- <b><a href="https://github.com/lh3/bioawk">bioawk</a></b> (ğŸ¥‰9 Â·  â­ 460 Â· ğŸ’€) - BWK awk modified for biological data.
- <b><a href="https://github.com/PacificBiosciences/pb-assembly">pb-assembly</a></b> (ğŸ¥‰9 Â·  â­ 89 Â· â•) - PacBio Assembly Tool Suite: Reads in Assembly out.
- <b><a href="https://github.com/refresh-bio/DSRC">DSRC</a></b> (ğŸ¥‰8 Â·  â­ 45 Â· â•) - DSRC - DNA Sequence Reads Compressor.
- <b><a href="https://github.com/babelomics/SMAca">SMAca</a></b> (ğŸ¥‰8 Â·  â­ 8 Â· ğŸ£) - SMN1 copy-number and sequence variant analysis from next generation sequencing..
- <b><a href="https://github.com/danielfan/MuSE">MuSE</a></b> (ğŸ¥‰6 Â·  â­ 14 Â· ğŸ’€) - Somatic point mutation caller.

